src/main/java/com/hivemq/cli/commands/cli/TestBrokerCommand.java

```diff
@@ -42,69 +37,84 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.tinylog.Logger;
-import org.tinylog.configuration.Configuration;
 import picocli.CommandLine;
 
 import javax.inject.Inject;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
+import java.util.Objects;
 
-@CommandLine.Command(
-        name = "test",
+@CommandLine.Command(name = "test",
         description = "Tests the specified broker on different MQTT feature support and prints the results",
         sortOptions = false)
 public class TestBrokerCommand implements Runnable {
 
-    final int MAX_PAYLOAD_TEST_SIZE = 100000; // ~ 1 MB
+    private static final int MAX_PAYLOAD_TEST_SIZE = 100000; // ~ 1 MB
 
-    @CommandLine.Option(names = {"-h", "--host"}, description = "The hostname of the message broker (default 'localhost')", order = 1)
-    private @Nullable String host;
+    private final @NotNull DefaultCLIProperties defaultCLIProperties;
 
+    private @Nullable MqttClientSslConfig sslConfig;
+
+    @SuppressWarnings("unused") //needed for pico cli - reflection code generation
+    public TestBrokerCommand() {
+        //noinspection ConstantConditions
+        this(null);
+    }
+
+    @Inject
+    public TestBrokerCommand(final @NotNull DefaultCLIProperties defaultCLIProperties) {
+        this.defaultCLIProperties = defaultCLIProperties;
+    }
```

Comment:
constructors come after fields

Reply:
Fixed for all classes now. Some command annotations are also bound to methods. In that case I treated those methods as fields and followed them with the backing fields.